cmd/broadcaster-cli/app/utxos/broadcast/broadcast.go

```diff
@@ -212,4 +218,10 @@ func init() {
 		log.Fatal(err)
 	}
 
+	Cmd.Flags().Int("size-jitter", 0, "Enable the option to randomise the transaction size, the parameter specifies the maximum size of bytes that will be added to OP_RETURN")
```

Comment:
Complete the description with information about the random number of inputs added to the transaction and specify that the jitter size only applies to the size of the added OP_RETURN and in some cases final added size can be greater due to test headers added to OP_RETURN

Reply:
Done